tests/unit/test__signing.py

```diff
@@ -655,16 +655,16 @@ def test_w_custom_query_parameters_w_none_value(self):
 
     def test_with_access_token(self):
```

Comment:
We need to test both the case where `access_token` is passed explicitly, but `service_account_email` is not (falling back to `client.signer_email`), *and* we need to test where both values are passed explicitly.  We also need assertions that the correct values are passed through to the `_sign_message` mock.

Reply:
Done, working path tested, not the exception cases. I can do it if required